packages/account-v2/src/containers/ProofOfIncome/ProofOfIncomeForm.tsx

```diff
@@ -0,0 +1,109 @@
+                                        <li className='py-5' key={`list-item-${item}`}>
+                                            <Text size='sm'>{item}</Text>
+                                        </li>
+                                    ))}
+                                </ul>
+
+                                <div className='flex flex-col items-start gap-16 w-full'>
+                                    <Text size='sm' weight='bold'>
+                                        Upload File
+                                    </Text>
+                                    <FormDocumentUploadField
+                                        className='w-full'
+                                        description='Remember, selfies , pictures of houses, or non-related images will be rejected.'
+                                        descriptionColor='primary'
+                                        descriptionSize='2xs'
+                                        fileFormats={[
+                                            'image/jpeg',
+                                            'image/jpg',
+                                            'image/png',
+                                            'image/gif',
+                                            'application/pdf',
+                                        ]}
+                                        hoverMessage='Upload your file here'
+                                        icon={
+                                            <StandaloneArrowUpFromBracketBoldIcon
+                                                className='fill-solid-green-2'
+                                                iconSize='lg'
+                                            />
+                                        }
+                                        maxSize={8388608}
+                                        name='document'
+                                        title='Drag and drop a file or click to browse your files.'
+                                        titleType='bold'
+                                        validationSchema={documentValidation}
+                                    />
+                                    <div className='flex items-start self-stretch justify-between'>
+                                        <Text size={isMobile ? '2xs' : 'xs'}>
+                                            Supported formats : JPEG, JPG, PNG, PDF, and GIF only
+                                        </Text>
+                                        <Text size={isMobile ? '2xs' : 'xs'}>Maximum size : 8MB</Text>
+                                    </div>
+                                </div>
+                            </section>
+                        </section>
+                    </section>
+                </Form>
+            </Formik>
+
+            <div>
+                <Divider />
+                <div className='sticky bottom-0 flex justify-end flex-shrink-0 w-full px-20 py-16 '>
```

Comment:
Lets avoid using absolute positioning. U can refer to `packages/account-v2/src/containers/POICountrySelector/POICountrySelector.tsx` to apply the same styles